Makefile

```diff
@@ -188,16 +188,25 @@ zip-wheels:
 
 ## Step: Upload wheels to pypi
 # Usage: TOKEN=... make upload-trulens-instrument-langchain
-upload-%: clean build zip-wheels
+# In all cases, we need to clean, build, zip-wheels, then build again. The reason is because we want the final build to have the zipped wheels.
+upload-%: clean build
+	make zip-wheels
+	make build
 	poetry run twine upload -u __token__ -p $(TOKEN) dist/$*/*
 
-upload-all: clean build zip-wheels
+upload-all: clean build
+	make zip-wheels
+	make build
 	poetry run twine upload --skip-existing -u __token__ -p $(TOKEN) dist/**/*.whl
 	poetry run twine upload --skip-existing -u __token__ -p $(TOKEN) dist/**/*.tar.gz
 
-upload-testpypi-%: clean build zip-wheels
+upload-testpypi-%: clean build
+	make zip-wheels
+	make build
 	poetry run twine upload -r testpypi -u __token__ -p $(TOKEN) dist/$*/*
 
-upload-testpypi-all: clean build zip-wheels
+upload-testpypi-all: clean build
+	make zip-wheels
```

Comment:
is this change due to the ordering the commands are run?

Reply:
I need to enforce the ordering, and also run build twice.